client/core/core.go

```diff
@@ -7075,6 +7092,10 @@ func (c *Core) findBondKeyIdx(pkhEqualFn func(bondKey *secp256k1.PrivateKey) boo
 // findBond will attempt to find an unknown bond and add it to the live bonds
 // slice and db for refunding later. Returns the bond strength if no error.
 func (c *Core) findBond(dc *dexConnection, bond *msgjson.Bond) (strength, bondAssetID uint32) {
+	if bond.AssetID == account.PrepaidBondID {
+		c.insertPrepaidBond(dc, bond)
+		return bond.Strength, bond.AssetID
```

Comment:
Any reason why we insert instead of checking db? I assume the redeemed prepaid bond was added to the db, no?